tests/Unit/FeatureTimeBombTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+declare(strict_types=1);
+
+use Carbon\Carbon;
+use JustSteveKing\Laravel\FeatureFlags\Models\Feature;
+use Illuminate\Support\Facades\Config;
+
+beforeEach(function(): void {
+    Config::set('feature-flags.enable_time_bombs', true);
+});
+
+afterAll(function(): void {
+    Config::set('feature-flags.enable_time_bombs', false);
+});
+
+it('can create a feature with an expiry date', function(): void {
+   Feature::create([
+       'name' => 'Expiring Feature',
+       'expires_at' => Carbon::now()->addDay()
+   ]);
+
+   expect(Feature::all())
+       ->toHaveCount(1);
+});
+
+it('Does not throw an Exception when time bombs are disabled', function(): void {
+    Config::set('feature-flags.enable_time_bombs', false);
+
+    Feature::create([
+        'name' => 'Expired Feature',
+        'expires_at' => Carbon::now()->subDay()
+    ]);
+
+    expect(Feature::all())->toHaveCount(1);
+    // Assert No Exception is thrown
+    $this->assertTrue(true);
```

Comment:
I think this is redundant since you already assert that the feature was created. If there had been an exception, the test would have failed before you got to this point in the test.

Reply:
Thank you, I've removed this test now.